requirements/pymeasure_dev.yml

```diff
@@ -0,0 +1,11 @@
+channels:
+  - conda-forge
+dependencies:
+  - pytest-qt=3.3.0
+  - pytest-runner=5.2
+  - pytest=6.1.1
+  - flake8=3.8.4
+  - sphinx=3.2.1
+  - sphinx_rtd_theme=0.4.3
+  - pip:
```

Comment:
[Apparently](https://github.com/pymeasure/pymeasure/pull/391/checks?check_run_id=1766645369#step:5:6) pip nowadays also has to be added as an explicit dependency.

Reply:
Thanks for adding that. I saw the funny error message, but did not finish adding it.